test/setup-beam.test.js

```diff
@@ -277,6 +285,222 @@ async function testOTPVersions() {
   simulateInput('hexpm-mirrors', hexMirrors, { multiline: true })
 }
 
+async function testLinuxARM64OTPVersions() {
+  let got
+  let expected
+  let spec
+  let osVersion
+  let before
+  const hexMirrors = simulateInput(
+    'hexpm-mirrors',
+    'https://repo.hex.pm, https://cdn.jsdelivr.net/hex',
+    { multiline: true },
+  )
+
+  const arm64Options = setupBeam.githubARMRunnerArchs()
+  process.env.RUNNER_ARCH =
+    arm64Options[Math.floor(Math.random() * arm64Options.length)]
+
+  if (process.platform === 'linux') {
+    before = simulateInput('version-type', 'strict')
+    spec = '26'
+    osVersion = 'ubuntu-24.04'
+    expected = 'maint-26'
+    got = await setupBeam.getOTPVersion(spec, osVersion, hexMirrors)
```

Comment:
I think this `hexMirrors` is wrong everywhere. Would you be so kind and remove it? (at least, from your changes, but also from all the other tests if it's Ok with you) Thanks.

Reply:
I'll merge as-is and do the fix, no worries.